.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Steps to Reproduce
+
+1. 
+2. 
+3. 
+
+## Expected Behavior
+
+Please describe the behavior you are expecting
+
+## Current Behavior
```

Comment:
nit: in most templates I've seen the current/problematic behavior comes before the expected one.